src/supremm/plugins/CgroupMemory.py

```diff
@@ -16,10 +16,15 @@ class SlurmCgroupMemory(Plugin):
     derivedMetrics = property(lambda x: [])
 
     def __init__(self, job):
-        super(SlurmCgroupMemory, self).__init__(job)
+        super(CgroupMemory, self).__init__(job)
         self._data = {}
         self._hostcounts = {}
-        self._expectedcgroup = "/slurm/uid_{0}/job_{1}".format(job.acct['uid'], job.job_id)
+        if job.acct['resource_manager'] == 'pbs':
+            self._expectedcgroup = "/torque/{0}".format(job.job_id)
```

Comment:
`job.job_id` is numeric and the value in PCP is actually the full job ID with batch server in string.

From `cgroup.memory.usage`
```
                inst [154 or "/torque/24.pitzer-batch.ten.osc.edu"] value 5154840576
```

Somehow the parsed jobid string from `Proc` preprocessor should be passed to the plugins?